src/util/psbt/error.rs

```diff
@@ -98,6 +102,8 @@ impl fmt::Display for Error {
             }
             Error::NoMorePairs => f.write_str("no more key-value pairs for this psbt map"),
             Error::HashParseError(e) => write!(f, "Hash Parse Error: {}", e),
+            Error::MissingUtxo => f.write_str("UTXO information is not present in PSBT"),
+            Error::PsbtUtxoOutOfbounds => f.write_str("Output index is out of bounds of non witness script output array"),
```

Comment:
Oh, forgot to comment before that `"Output index` should probably be `"output index` to be consistent with other messages.

Reply:
`Hash Parse Error:` is wrong too, a patch put at the front of this PR cleaning that all up would be super useful if you can be bothered. Don't feel obliged though. You will likely touch this line when the error comment is changed anyways.